backends/arm/quantizer/arm_quantizer.py

```diff
@@ -357,9 +368,30 @@ def _annotate_all_static_patterns(
             OP_TO_ANNOTATOR[op](model, quantization_config, filter_fn)
         return model
 
+    def _annotate_io(
```

Comment:
> The intended use is for unit testing
quantization of operations
without quantization annotators.

This is interesting. I assume this also generates q->dq nodes for IOs?

Reply:
Yes, what I want to achieve is
in -> q1 -> dq1 -> op -> q2 -> dq2 -> out

Reply:
I guess quantizing the `op` should get you the same graph?

Reply:
Yes, I just wanted a quant flow that is similar to how the operation would be quantized "in the wild", which is by getting a qspec propagated from the parent.

 (A rebase is on its way, I'm just away on vacation next week and then some training)